spec/features/projects/persisted_lists_sharing_spec.rb

```diff
@@ -169,7 +174,8 @@
 
           # Can save the project list
           projects_index_page.save_query
-          projects_index_page.expect_toast(message: "The modified list has been saved")
+          # TODO: Toast is currently not rendered in turbo actions
+          # projects_index_page.expect_toast(message: "The modified list has been saved")
```

Comment:
Also enable this? https://github.com/opf/openproject/pull/16246/files/8e82fe6167102988a914d1702ab16b7b20941e85#r1724915608

Reply:
Yes.

Reply:
Do you have any idea why the toast is sent as a stream properly but isn't on the test suite? I believe Klaus mentioned there was some funny behavior with the toast in the specs. There should be a PR Oliver and Mir are working on for a better way to deal with this that SHOULD resolve this weird behavior in specs. @dombesz

Reply:
It's been a while ago, but I think there was something problematic because the way how the toast javascript was triggered was that it only worked on the page load and it did not properly work with turbo. But I'm not 100% sure where this was